.github/workflows/generate-event-pdfs.yml

```
@@ -2,7 +2,18 @@ name: generate-pdf-artifacts
 
 on:
   workflow_dispatch:
-    inputs: {}
+    inputs:
+      REMOVE_MASTER:
+        description: Remove the master files
+        required: false
+        default: '1'
```

To be consistent, although it's likely represented as a shell environment variable (and you can't tell between `'1'` and `1` it would still be good to be consistent here, and not have what appears to be a string in one place, an integer in the other.
```suggestion
        default: 1
```